fe/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -224,7 +225,17 @@ private void processDropColumn(DropColumnClause alterClause, OlapTable olapTable
                 }
             }
         }
-        
+
+        Iterator<Index> it = indexes.iterator();
+        while(it.hasNext()){
+            Index index = it.next();
+            if(index.getIndexType() == IndexDef.IndexType.BITMAP
```

Comment:
If the column is dropped, all type of index should be dropped, so here should not add this if clause.